Ktisis/Structs/Havok/HkaPose.cs

```diff
@@ -5,5 +5,7 @@ namespace Ktisis.Structs.Havok {
 	public unsafe struct HkaPose {
 		[FieldOffset(0x00)] public HkaSkeleton* Skeleton;
 		[FieldOffset(0x10)] public ShitVecReversed<Transform> Transforms;
+		[FieldOffset(0x38)] public byte ModelInSync;
+		[FieldOffset(0x39)] public byte LocalInSync;
```

Comment:
These look unused. Can you split them into a separate PR if they're not needed yet?

Reply:
Same as above.